pkgs/by-name/mu/muse-sounds-manager/package.nix

```diff
@@ -14,33 +13,36 @@
   libICE,
   libSM,
   openssl,
+  unzip,
+  xdg-utils,
+  makeWrapper,
 }:
 
 stdenv.mkDerivation rec {
   pname = "muse-sounds-manager";
-  version = "1.1.0.587";
+  version = "2.0.4.872";
 
   # Use web.archive.org since upstream does not provide a stable (versioned) URL.
   # To see if there are new versions on the Web Archive, visit
-  # http://web.archive.org/cdx/search/cdx?url=https://muse-cdn.com/Muse_Sounds_Manager_Beta.deb
+  # http://web.archive.org/cdx/search/cdx?url=https://muse-cdn.com/Muse_Sounds_Manager_x64.tar.gz
   # then replace the date in the URL below with date when the SHA1
-  # changes (currently A3NX3WHFZWXCHZVME2ABUL2VRENTWOD5) and replace
+  # changes (currently QLR46LKDOAPB7VSF45HEAXWVNWFJHITG) and replace
   # the version above with the version in the .deb metadata (or in the
   # settings of muse-sounds-manager).
   src = fetchurl {
-    url = "https://web.archive.org/web/20240826143936/https://muse-cdn.com/Muse_Sounds_Manager_Beta.deb";
-    hash = "sha256-wzZAIjme1cv8+jMLiKT7kUQvCb+UhsvOnLDV4hCL3hw=";
+    url = "https://web.archive.org/web/20250729165100if_/https://muse-cdn.com/Muse_Sounds_Manager_x64.tar.gz";
+    hash = "sha256-VcLBXpLDk90yd0j9NIzBOXXAciSLWP9y5X51L2/9W4A=";
   };
 
   nativeBuildInputs = [
     autoPatchelfHook
-    dpkg
   ];
 
   buildInputs = [
     fontconfig
     stdenv.cc.cc
     zlib
+    makeWrapper
```

Comment:
makeWrapper goes into nativeBuildInputs

Reply:
Thank you for the review. The suggestion has been applied.